src/components/code-snippet/multi-line-code-snippet.html

```diff
@@ -0,0 +1,40 @@
+<div class="bx--snippet bx--snippet--multi">
```

Comment:
Great to add `data-code-snippet` here (and change `selectorInit` option accordingly) to make our JavaScript doesn’t have to deal with changes in CSS class structure. This means the root element for our JavaScript code is the whole snippet instead of the expando. Doing so allows us to remove an assumption on the DOM structure - The expando being a direct child of the code snippet.